build_container/Dockerfile-mobile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:focal
+
+# Workaround for: https://issuetracker.google.com/issues/216325949
+ENV CLOUDSDK_PYTHON=python3
+
+COPY ./build_container_common.sh /
+COPY ./build_container_ubuntu.sh /
```

Comment:
im wondering if we need llvm in this image - if not can we skip it somehow ?

Reply:
We run asan/tsan tests which I believe need LLVM installed.

Reply:
we may be able to thin out some of the other deps for this also i guess - not sure if its worth the effort of splitting out tho - i think its llvm that takes the majority of the space (iirc)

Reply:
For now, I'd rather not risk removing something we don't know we rely on, and tackle further size reductions as a follow-up.